nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkService.java

```diff
@@ -18,16 +18,26 @@
  */
 package org.nuxeo.ecm.core.bulk;
 
-import java.util.UUID;
-
 /**
- * API to manage Bulk Operation Framework.
+ * API to manage Bulk Action Framework.
  *
  * @since 10.2
  */
 public interface BulkService {
 
-    BulkStatus runOperation(BulkCommand command);
+    /**
+     * Submits the input {@link BulkCommand} to the Bulk Action Framework system and get the initial status containing
+     * the bulk action id.
```

Comment:
We need stricter definition of things:
- terms like "input" and "system" are just noise
- a service that manages a framework is weird, may be keep BAF at documentation level and stick to bulk action at code level ?
- are you running an action or a command ?
- submit may be better name than runAction because of the asynchronous processing

Reply:
Agree with you for naming.
I will remove input and system, and remove framework from our code.

For "running an action or a command", indeed it's not clear, here some description: we submit a command, then an action (`SetPropertiesAction` for instance) does the work on the scrolled set.

Submit is good for method name. With that, we submit a command, then we scroll DB, then an action does computation. I don't think there's ambiguity anymore, is it ?

Remaining weird thing is places where we talk about bulk action id, which is in fact the bulk id (the one present in bulk status). We should rename bulkActionId to bulkId.